parts/windows/csecmd.ps1

```diff
@@ -1 +1,20 @@
-"[concat('echo %DATE%,%TIME%,%COMPUTERNAME% && powershell.exe -ExecutionPolicy Unrestricted -command \"', '$arguments = ', variables('singleQuote'),'-MasterIP ',parameters('kubernetesEndpoint'),' -KubeDnsServiceIp ',parameters('kubeDnsServiceIp'),' -MasterFQDNPrefix ',variables('masterFqdnPrefix'),' -Location ',variables('location'),' -TargetEnvironment ',parameters('targetEnvironment'),' -AgentKey ',parameters('clientPrivateKey'),' -AADClientId ',variables('servicePrincipalClientId'),' -AADClientSecret ',variables('singleQuote'),variables('singleQuote'),base64(variables('servicePrincipalClientSecret')),variables('singleQuote'),variables('singleQuote'),' -NetworkAPIVersion ',variables('apiVersionNetwork'),' ',variables('singleQuote'), ' ; ', variables('windowsCustomScriptSuffix'), '\" > %SYSTEMDRIVE%\\AzureData\\CustomDataSetupScript.log 2>&1 ; exit $LASTEXITCODE')]"
\ No newline at end of file
+echo %DATE%,%TIME%,%COMPUTERNAME% && powershell.exe -ExecutionPolicy Unrestricted -command \"
```

Comment:
This is referenced from
https://github.com/Azure/aks-engine/blob/aks-release-v0.47.0-1/pkg/engine/virtualmachinescalesets.go#L794.
The  changed code is more readable

Reply:
This script  is not used, but kept for further reference.

Reply:
I assume you will file a second PR to use this template?

Reply:
do you plan to use agentbaker cse in ARM template and then convert ARM template to vmss-client for windows?
or do you plan to use agentbaker cse directly in vmss-client? 
My imagination is we go with the latter one and if so, this file should be used

Reply:
We plan to first use agentbaker cse in ARM template and then convert ARM template to vmss-client for windows?
Just to be careful about the change. Also, I hit some issue when passing certificate in cse command, needing more time to fix.

Reply:
This file will definitely be used in our further PR.

Reply:
Got you. Thanks!